generators/app/index.js

```diff
@@ -27,65 +33,76 @@ module.exports = Generator.extend({
         name: 'githubUsername',
         message: 'What is the github project organisation or username?',
         validate: required,
-        default: githubUsername
+        default: githubUsername,
+        when: utils.noConfig('githubUsername', this.initialConfig)
```

Comment:
The problem is when `0, 1, false, true` are saved
`!0` evaluates to true, the prompt is execute
`!1` evaluates to false, the prompt is not executed

That is way checking for the existence of the key behaves like it should (true in the two cases).
The correct is `this.initialConfig[''githubUsername"] == null || this.initialConfig["githubUsername"] == undefined`. Which I think `utils.noConfig('githubUsername', this.initialConfig)` is more readable.

What do you think?